js/app.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
+
+// Add listener.
+  $("#submit-btn").click(doStuff);
+
+// Create function for clicking submit button.
+  function doStuff() {
+    console.log("clicked submit");
+  
+  // Read data typed into text field.
+  // Store it in a variable.
+
+    var city = $("#city-type").val();
```

Comment:
@leebovel Consider converting the value of the input to the same case of the what you are validating